pkgs/by-name/ss/sstp/package.nix

```
@@ -34,14 +36,20 @@ stdenv.mkDerivation {
   nativeBuildInputs = [
     pkg-config
     autoreconfHook
+    makeWrapper
```

```suggestion
    makeBinaryWrapper
```
Always use `makeBinaryWrapper` unless you have a good reason not to. It's more robust than the bash script created by `makeWrapper`, as it generates and compiles C code.